docs/aurora/data-science/frameworks/pyg.md

```diff
@@ -0,0 +1,177 @@
+#  PyTorch Geometric (PyG)
+
+[PyTorch Geometric (PyG)](https://pyg.org/) is a Python library built on top of PyTorch for deep learning on graphs. 
+It provides tools for working with graph-structured data, and implementations of many Graph Neural Networks (GNNs). 
+
+
+## PyG on Aurora
+
+PyTorch Geometric includes a base library, called [`torch_geometric`](https://github.com/pyg-team/pytorch_geometric), and a number of optional dependencies (
+[`torch_scatter`](https://github.com/rusty1s/pytorch_scatter), 
+[`torch_sparse`](https://github.com/rusty1s/pytorch_sparse), 
+[`torch_cluster`](https://github.com/rusty1s/pytorch_cluster), 
+[`torch_spline_conv`](https://github.com/rusty1s/pytorch_spline_conv), 
+[`pyg_lib`](https://github.com/pyg-team/pyg-lib) 
+), which include optimized operations that are relevant to GNNs and are missing in PyTorch. 
+
+The [base library](#pyg-base-library), `torch_geometric`, relies solely on PyTorch and can utilize Intel GPUs through the `xpu` device specification. 
+
+The [optional dependencies](#optional-dependencies) have only CPU and CUDA implementations and can therefore only run on Aurora CPUs.
+
+
+## PyG base library 
+
+Here we explain how to install the base library, `torch_geometric`, on Aurora and show a simple example of training a PyG model on Intel GPUs. 
+
+
+### Installing `torch_geometric` on Aurora
+
+We are going to install `torch_geometric` in a [virtual environment](../python.md#virtual-environments-via-venv) that inherits PyTorch and the other libraries from the base frameworks module:
+
+```bash
+module load frameworks
+python3 -m venv --clear venv --system-site-packages
+source venv/bin/activate
+pip install torch_geometric
+```
+
+
+### Example
+
+The following example is inspired by the [`gcn.py` example](https://github.com/pyg-team/pytorch_geometric/blob/master/examples/gcn.py) on the PyG repository. 
+
+- Copy the following Python script into a  file called `gcn.py`.
+  #### `gcn.py`
```

Comment:
better to use code block titles instead of section heads

https://squidfunk.github.io/mkdocs-material/reference/code-blocks/#adding-a-title